store/src/sql/entities/query.rs

```diff
@@ -34,8 +36,21 @@ impl EventQuery {
             ORDER BY eb.idx;"#
     }
 
+    /// Requires binding 1 parameter. Finds the `EventValueRaw` values needed to rebuild the event
+    /// Looks up the event by the root CID of the event.
+    /// TODO(stbrody): Does it need to return order_key at all?  Can the join on event table be removed?
```

Comment:
It needs to return whatever you want it to be 😄.. right now it's deserialized into this struct so it does need order_key, but if we can remove the need for event ID that can maybe be dropped (not sure since recon relies so heavily on it)

```
#[derive(Debug, Clone, sqlx::FromRow)]
pub struct EventValueRaw {
    pub order_key: Vec<u8>,
    pub codec: i64,
    pub root: bool,
    pub idx: i32,
    pub multihash: Vec<u8>,
    pub bytes: Vec<u8>,
}```

Reply:
Where is it being used to populate an `EventValueRaw`?  I only see it being turned into a `BlockRow`

Reply:
Oh, I think you're right.. so they are different queries but it's these two returning basically the same columns.

```rust
let blocks: Vec<BlockRow> = sqlx::query_as(EventQuery::value_blocks_one()) // value_blocks_by_order_key_one
let all_blocks: Vec<EventBlockRaw> = sqlx::query_as(EventQuery::value_blocks_many()) // value_blocks_by_cid_one
```